app/helpers/application_helper/button/cockpit_console.rb

```diff
@@ -13,4 +14,8 @@ def on?
     return @record.current_state == 'on' if @record.respond_to?(:current_state) # VM status
     @record.ready_condition_status == 'True' if @record.respond_to?(:ready_condition_status) # Container status
   end
+
+  def platform_supported?
+    @record.platform.downcase != 'windows' if @record.respond_to?(:current_state)
```

Comment:
@mzazrivec, I'm checking if it responds to current state to verify it's a VM so that the platform can be checked.  In my research I was unable to determine a reliable way to verify the platform of a Container Node.